src/Tasks/SystemState.cs

```diff
@@ -71,7 +72,7 @@ internal sealed class SystemState : StateFileBase, ITranslatable
         /// <summary>
         /// True if the contents have changed.
         /// </summary>
-        private bool isDirty;
+        internal bool isDirty;
```

Comment:
nit: Still need `internal` access? Isn't it a left-over from when the cache was Json-serialized? Same for the other `private` -> `internal` changes.

Reply:
Actually, this (and others) were for testing so I could make a cache and serialize it without modifying it or actually executing RAR.

Reply:
Gotcha. This one can be private, though, as it has an internal read/write property, is that correct?

Reply:
Yep! You're right. I added the write part of that property, so it seems I gave myself one way to access it, forgot, and gave myself a second way to access it. Fixed!